circle.yml

```diff
@@ -1,16 +1,19 @@
 machine:
+  node:
+    version: 8.4.0
```

Comment:
We need to use node >=8.4.0 for `npm install -g sfdx-cli` to work. I think this is fine since the LWC tests also require this and I can foresee LTS also using a new version.